### ISSUE TITLE
FR: Add Copy link to asset manager

### ISSUE THREAD

**Issue by mikemartin** [statamic/v2-hub#1360]
There seems to be no way to copy the URL of an asset. Could we add a Copy Link action on the Asset listing dropdowns and asset editor actions?

<img width="887" alt="screenshot 2017-03-29 13 45 32" src="https://cloud.githubusercontent.com/assets/414211/24468374/013cc8a8-1486-11e7-9c01-1aa86421b206.png">

![screenshot 2017-03-29 13 49 16](https://cloud.githubusercontent.com/assets/414211/24468533/8a42c990-1486-11e7-9f05-14a1cad1eaca.png)

**Comment by anthubbard** [comment]
Can I please +1 this. And maybe +2. Got some annoyed clients cause copying assets is currently a PITA and doesn't fit the simple awesomeness that is the rest of the Statamic experience.

**Comment by rockethouse** [comment]
Yeah just had a client ask about this and I confidently said it was possible in the Asset library before being embarrassed and found it was missing... definite must have!
